praw/exceptions.py

```diff
@@ -112,21 +138,86 @@ def _get_old_attr(self, attrname):
 
     def __init__(
         self,
-        items: Union[List[Union[RedditErrorItem, List[str], str]], str],
+        items: Union[
+            List[Union[RedditErrorItem, List[str], str]], str, RedditErrorItem
+        ],
         *optional_args: str
     ):
         """Initialize an instance of RedditAPIException.
 
-        :param items: Either a list of instances of :class:`.RedditErrorItem`
-            or a list containing lists of unformed errors.
-        :param optional_args: Takes the second and third arguments that
-            :class:`.APIException` used to take.
+        :param items: The items representing the APIException. Can be one of
+            several things.
+
+                * A list of instances of :class:`.RedditErrorItem`.
+                * A list of two to three of the strings needed to construct an
+                  instance of :class:`.RedditErrorItem`.
+                * A list containing lists with the items mentioned in the
+                  second bullet.
+                * A string representing the first of the two to three of the
+                  strings needed to construct an instance of
+                  :class:`.RedditErrorItem`. In that case, parameter
+                  ``optional_args`` needs to be provided.
+                * An instance of :class:`.RedditErrorItem`.
+
+        :param optional_args: Takes the second and/or third arguments that
+            :class:`.APIException` used to take. Also represents the second and
```

Comment:
Can we remove this `used to take` part? For people reading this documentation in the future, that might not make sense. Do we actually need that support any more, or is that just for backwards compatibility? If the latter, we definitely don't need to document it.

Ideally we only support the methods that PRAW uses internally to create this exception, and we should try to normalize at call-time those as much as possible to simplify this class.